lib/Froxlor/Api/Commands/Mysqls.php

```diff
@@ -31,13 +31,15 @@ class Mysqls extends \Froxlor\Api\ApiCommand implements \Froxlor\Api\ResourceEnt
 	 *        	optional, default is 0
 	 * @param string $description
 	 *        	optional, description for database
+	 * @param string $name
```

Comment:
the variable-name `$name` should be equal the `getParam('variable-name')` as the automated API documentation takes its info of the phpDoc.

Also `database_name` could be irritating, maybe we should use something like `custom_suffix`? 

phpDoc should then be
`@param string $custom_suffix`
and to get the parameter:
`$databasename = $this->getParam('custom_suffix', true, '');`

Reply:
I changed it to custom_suffix =)